static/app/views/alerts/rules/metric/create.spec.tsx

```diff
@@ -46,6 +47,25 @@ describe('Incident Rules Create', function () {
 
     render(
       <MetricRulesCreate
+        {...TestStubs.routeComponentProps()}
+        eventView={
+          new EventView({
+            id: '',
+            createdBy: TestStubs.User(),
+            display: '',
+            project: [],
+            environment: [],
+            name: '',
+            query: '',
+            fields: [],
+            sorts: [],
+            statsPeriod: '14d',
+            team: [],
+            topEvents: undefined,
+            start: '',
+            end: '',
+          })
```

Comment:
i think this is the event we use when creating https://github.com/getsentry/sentry/blob/master/static/app/views/alerts/create.tsx#L109
```suggestion
          EventView.fromLocation(TestStubs.routeComponentProps().location)
```